quotes.js

```diff
@@ -0,0 +1,16 @@
+let quotes = document.getElementById("quote");
+let author = document.getElementById("author");
+let tags = document.getElementById("tag-item");
+
+async function getQuotes() {
+  let result = await fetch("https://api.quotable.io/random");
+  let resultJson = await result.json();
+
+  let content = resultJson.content;
+  let writer = resultJson.author;
```

Comment:
This is totally fine, Just a simple tip. You can change this to:
```js
const {content , writer} = resultJson;
```

Reply:
or if you want to go even further:
```js
const {content , writer} = await result.json();
```

Reply:
or even more (don't do that).
```js
const {content , writer} = await (await fetch("https://api.quotable.io/random")).json();
```

Reply:
Done.